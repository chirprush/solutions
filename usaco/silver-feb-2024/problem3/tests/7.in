1
20 5 3
2 2 15
1 1 1
1 1 1
7 6 7
1 1 1
