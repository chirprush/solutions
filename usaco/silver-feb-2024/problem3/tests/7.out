Even Odd Odd Odd Odd
